pkg/vulnerabilityreport/controller/workload.go

```diff
@@ -252,19 +252,34 @@ func (r *WorkloadController) SubmitScanJob(ctx context.Context, owner client.Obj
 		"name", owner.GetName(), "namespace", owner.GetNamespace())
 	var err error
 	credentials := make(map[string]docker.Auth, 0)
+
+	privateRegistrySecrets, err := r.Config.GetPrivateRegistryScanSecretsNames()
+	if err != nil {
+		return err
+	}
+
+	pConfig, err := r.PluginContext.GetConfig()
+	if err != nil {
+		return err
+	}
+
+	multiSecretSupport := trivy.MultiSecretSupport(trivy.Config{PluginConfig: pConfig})
```

Comment:
also I have a concern about this block.
should we run it for existed KBOR reports?